### ISSUE TITLE
Feature Request - Get DAC Voltage

### ISSUE THREAD

**Issue by mserlin** [afylab/modularElectronics#11]
The code could internally keep track of the latest voltage that was set on each output channel. We then could read out the values with a function. Is there anything wrong with this?

**Comment by Jehrets2** [comment]
No, that would be a good work around. However it would be nice to have the functionality to check on the DAC with the proper read register command, which I am now working on. Will update.
